PWGDQ/TableProducer/tableMaker.cxx

```diff
@@ -1334,7 +1334,13 @@ struct TableMaker {
   {
     fullSkimming<gkEventFillMap, 0u, gkMuonFillMapWithCov>(collision, bcs, nullptr, tracksMuon, nullptr, nullptr);
   }
-
+                                              
+  // Produce muon tables only, with muon cov matrix and multiplicity --------------------------------------------------------------------------------------------
+  void processMuonOnlyWithCovMults(MyEventsWithMults::iterator const& collision, aod::BCsWithTimestamps const& bcs,
```

Comment:
@bhsaho To be consistent with the names of the other process functions, could you call it **processMuonOnlyWithCovAndMults** ?

Reply:
Okay

Reply:
Is it fine now ?